.buildkite/react-native-pipeline.yml

```diff
@@ -59,6 +59,28 @@ steps:
 
   - wait
 
+  - label: ':android: Build react-navigation 0.63 apk'
+    env:
+      REACT_NATIVE_VERSION: "rn0.63"
```

Comment:
I think we also need to build for RN 0.60?

Reply:
Yes, this is more a test run than anything else atm